src/main/g8/commit-with-git-before-sbt.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I'd rather not have a script for this. Especially things that commit on the user's behalf. This is in the `README` [already](https://github.com/hmrc/hmrc-frontend-scaffold.g8#creating-a-new-service).